### ISSUE TITLE
How can I export multiple animations(actions)？

### ISSUE THREAD

**Issue by lishengzxc** [KhronosGroup/glTF-Blender-Exporter#112]
![](https://gw.alicdn.com/tfs/TB1_AfocLDH8KJjy1XcXXcpdXXa-1436-877.jpg)

I have three actions. When I export model, I only have one action(currently action "Attack").

And Animtion has no name.

![](https://gw.alicdn.com/tfs/TB1I8sqaiqAXuNjy1XdXXaYcVXa-288-171.png)

![](https://gw.alicdn.com/tfs/TB1OCEqaiqAXuNjy1XdXXaYcVXa-686-339.png)

How can I export three animations(actions)?

**Comment by lishengzxc** [comment]
https://github.com/KhronosGroup/glTF-Blender-Exporter/pull/111

@cdata Your PR can solve it?

**Comment by cdata** [comment]
@lishengzxc yes, it does give you a solution to access the animations in the exported file. If you are using the GLTFLoader, you should be able to access the animations with the patch in my PR.

**Comment by KunyiLockeLin** [comment]
I just meet the same problem. So blender can exports only one action to gltf. Is it correct? It can't export plural actions. Right?

**Comment by donmccurdy** [comment]
> So blender can exports only one action to gltf. Is it correct? It can't export plural actions. Right?

That is a correct description of the problem. Sorry it isn't fixed yet.

**Comment by IMCAG** [comment]
Is there any waay to estimate when this issue going to be fixxed?
Any schedule for the corresponding milestone?

**Comment by IARI** [comment]
Is there any waay to estimate when this issue going to be fixxed?
Any schedule for the corresponding milestone?

**Comment by donmccurdy** [comment]
There is a work-in-progress solution here: https://github.com/KhronosGroup/glTF-Blender-Exporter/pull/166 and you can download the experimental exporter from that thread. I don't have an estimate of when the PR will be merged.

**Comment by ladiszlai** [comment]
Any updates when it will be implemented to the exporter ?

**Comment by donmccurdy** [comment]
@ladiszlai see above; you are welcome to try the work-in-progress version. Please report issues there if it doesn't work. I don't have an estimate of when the PR will be merged, I have very limited time but others are welcome to suggest PRs as well. :)

**Comment by donmccurdy** [comment]
It is not stable enough; some things would break if it were merged now.

**Comment by titansoftime** [comment]
Considering three.js just dumped its json exporter, this would be nice to have working.

My workflow is now in limbo.

**Comment by donmccurdy** [comment]
@titansoftime https://github.com/KhronosGroup/glTF-Blender-Exporter/pull/166 is a reasonably functional WIP. That may work for you; if not, letting us know of any issues would help to move it along.

**Comment by titansoftime** [comment]
Right on, will check it out. Thank you.

**Comment by olluciano** [comment]
i will try the exporter too, just bumped into same problem, im also using threejs and multiple actions
